src/pages-and-resources/data/api.js

```diff
@@ -1,61 +1,39 @@
 /* eslint-disable import/prefer-default-export */
-export function getPages() {
-  return Promise.resolve({
-    pages: [
-      {
-        id: 'discussions',
-        title: 'Discussions',
-        isEnabled: false,
-        showSettings: false,
-        showStatus: false,
-        showEnable: true,
-        description: 'Encourage participation and engagement in your course with discussion forums.',
-      },
-      {
-        id: 'teams',
-        title: 'Teams',
-        isEnabled: true,
-        showSettings: true,
-        showStatus: true,
-        showEnable: false,
-        description: 'Leverage teams to allow learners to connect by topic of interest.',
-      },
-      {
-        id: 'progress',
-        title: 'Progress',
-        isEnabled: false,
-        showSettings: true,
-        showStatus: true,
-        showEnable: false,
-        description: 'Allow students to track their progress throughout the course lorem ipsum.',
-      },
-      {
-        id: 'textbooks',
-        title: 'Textbooks',
-        isEnabled: true,
-        showSettings: true,
-        showStatus: true,
-        showEnable: false,
-        description: 'Provide links to applicable resources for your course.',
-      },
-      {
-        id: 'notes',
-        title: 'Notes',
-        isEnabled: true,
-        showSettings: true,
-        showStatus: true,
-        showEnable: false,
-        description: 'Support individual note taking that is visible only to the students.',
-      },
+import { ensureConfig, getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+ensureConfig([
+  'STUDIO_BASE_URL',
+], 'Course Apps API service');
+
+const apiBaseUrl = getConfig().STUDIO_BASE_URL;
+
+const courseAppsApiUrl = `${apiBaseUrl}/api/course_apps/v1/apps`;
+
+/**
+ * Fetches the course apps installed for provided course
+ * @param {string} courseId
+ * @returns {Promise<[{}]>}
+ */
+export async function getCourseApps(courseId) {
+  const { data } = await getAuthenticatedHttpClient()
+    .get(`${courseAppsApiUrl}/${courseId}`);
+  return data;
+}
```

Comment:
I can't seem to find if this data is being normalized here (yet), Do you have plans to normalize this data in this PR?

Reply:
There isn't much to normalise here. The data is pretty-much ready to use.

Reply:
I think we can normalize `allowed_oprations` here.